tools/ci_build/github/azure-pipelines/templates/c-api-cpu.yml

```diff
@@ -94,11 +94,15 @@ stages:
       buildSettings: '$(Build.SourcesDirectory)/tools/ci_build/github/android/default_qnn_aar_build_settings.json'
       artifactName: 'onnxruntime-android-qnn-aar'
       job_name_suffix: 'QNN'
-      publish_executables: '0'
+      publish_executables: '1'
```

Comment:
why is `publish_executables` set? I think with this option:
https://github.com/microsoft/onnxruntime/blob/08cc2612f46484030ad92ddf0baac9ab3fae0f6c/tools/ci_build/github/android/default_qnn_aar_build_settings.json#L15

there won't be anything interesting to publish (e.g., onnxruntime_perf_test won't get built).